src/software/firmware/srcs/respirator.cpp

```diff
@@ -138,6 +138,10 @@ void setup() {
 // Time of the previous loop iteration
 int32_t lastMicro = 0;
 
+// Number of cycles before LCD screen reset
+// (because this kind of screen is not reliable, we need to reset it every 5 min or so)
+int8_t cyclesBeforeScreenReset = LCD_RESET_PERIOD * CONST_MIN_CYCLE;
```

Comment:
With a high number cycles per minute, we can have up to a reset every 30s. I was wondering if it can be annoying for the user if the flickering is really visible.